roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1163,6 +1167,49 @@ def init_local_facts(self, facts=None):
         self.changed = changed
         return new_local_facts
 
+    def init_in_docker_facts(self, facts):
+        facts['is_containerized'] = os.path.isfile('/run/ostree-booted')
```

Comment:
If a general fact, we should have it under common, and since we want users to be able to run containerized outside of atomic, that would let them override at the playbook level.